device/accton/x86_64-accton_as7712_32x-r0/minigraph.xml

```diff
@@ -0,0 +1,75 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
stcheng,
YES currently~ No regular front panel interfaces in our minigraph file.
We'll update it once we clarify your minigraph configuration model by more testing at our side.